packaging/rpm/docker/rhel9/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM registry.access.redhat.com/ubi9
+
+# Enable ZeroC third-party repository for mcpp-devel
+RUN curl -o /etc/yum.repos.d/zeroc-thirdparty.repo https://builds.testing.zeroc.com/public/dist/unstable/thirdparty/el9/zeroc-thirdparty.repo
```

Comment:
This is not going to work in CI. Maybe we should also build the third party rpms as well, or build them in and have them installed in the container already.

Reply:
I'm uploading them to 3.7 public repository, they are required to build 3.7 from sources and are missing there.